extensions/file/README.md

```diff
@@ -14,19 +14,42 @@ Provides a way to specify file related details such as checksum, data type and s
 
 ## *Link Object* and *Asset Object* fields
 
-The following fields can be used for Links (in the [`Link Object`](../../item-spec/item-spec.md#link-object)) and assets (in the [`Asset Object`](../../item-spec/item-spec.md#asset-object)).
-
-| Field Name         | Type   | Description                                                  |
-| ------------------ | ------ | ------------------------------------------------------------ |
-| file:byte_order | string | The byte order of integer values in the file. One of `big-endian` or `little-endian`. |
-| file:checksum | string | Provides a way to specify file [checksums](#checksums) (e.g. BLAKE2, MD5, SHA1, SHA2, SHA3). The hashes are self-identifying hashes as described in the [Multihash specification](https://github.com/multiformats/multihash) and must be encoded as hexadecimal (base 16) string with lowercase letters. |
-| file:data_type | string | The data type of the file. One of the [data types](#data-types) below. |
-| file:header_size | integer | The header [size](#sizes) of the file, specified in bytes. |
-| file:size | integer | The file [size](#sizes), specified in bytes. |
+The following fields can be used for Links (in the [`Link Object`](../../item-spec/item-spec.md#link-object)) and assets (in the [`Asset Object`](../../item-spec/item-spec.md#asset-object)) although most of these fields make more sense for assets and are usually not very useful for links.
+
+| Field Name           | Type                                    | Description                                                  |
+| -------------------- | --------------------------------------- | ------------------------------------------------------------ |
+| file:bits_per_sample | integer                                 | Bits per sample, e.g. 8, 16, 32, ...                         |
+| file:byte_order      | string                                  | The byte order of integer values in the file. One of `big-endian` or `little-endian`. |
+| file:checksum        | string                                  | Provides a way to specify file [checksums](#checksums) (e.g. BLAKE2, MD5, SHA1, SHA2, SHA3). The hashes are self-identifying hashes as described in the [Multihash specification](https://github.com/multiformats/multihash) and must be encoded as hexadecimal (base 16) string with lowercase letters. |
+| file:data_type       | string                                  | The data type of the file. One of the [data types](#data-types) below. |
+| file:header_size     | integer                                 | The header [size](#sizes) of the file, specified in bytes.   |
+| file:nodata          | \[any]                                  | Value(s) for no-data.                                        |
```

Comment:
```suggestion
| file:nodata          | \[any\]                                  | Value(s) for no-data.                                        |
```
Do you need a closing `\]`?

Reply:
No, but it doesn't make any harm either, right?

Reply:
I was just confused because you included a closing `\]` a few lines below.

Reply:
Yeah, when writing by hand you sometimes do weird things ;-)